### ISSUE TITLE
[Stripe - NS7 + Angular] Error compiling Expiry

### ISSUE THREAD

**Issue by salvatorebnt** [triniwiz/nativescript-plugins#30]
Nativescript 7.1.0
Angular 11
Hello, i am using CreditCardView in my application, to catch the card info by the user:
Having an exception when i insert data in the Expiry text field, using the latest version of the plugin (@triniwiz/nativescript-stripe": "^7.0.0-alpha7")
Here the fatal exception:

```
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onTextChanged failed
System.err: TypeError: param0.split is not a function
System.err: 
System.err: StackTrace:
System.err: onTextChanged(file: node_modules\@triniwiz\nativescript-stripe\index.android.js:785:0)
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err:     at com.tns.gen.android.text.TextWatcher.onTextChanged(TextWatcher.java:31)
System.err:     at android.widget.TextView.sendOnTextChanged(TextView.java:10578)
System.err:     at android.widget.TextView.handleTextChanged(TextView.java:10685)
System.err:     at android.widget.TextView$ChangeWatcher.onTextChanged(TextView.java:13568)
System.err:     at android.text.SpannableStringBuilder.sendTextChanged(SpannableStringBuilder.java:1267)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:576)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:507)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:37)
System.err:     at android.text.method.NumberKeyListener.onKeyDown(NumberKeyListener.java:131)
System.err:     at android.widget.TextView.doKeyDown(TextView.java:8466)
System.err:     at android.widget.TextView.onKeyDown(TextView.java:8240)
System.err:     at android.view.KeyEvent.dispatch(KeyEvent.java:2842)
System.err:     at android.view.View.dispatchKeyEvent(View.java:14247)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.widget.ScrollView.dispatchKeyEvent(ScrollView.java:491)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:476)
System.err:     at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1861)
System.err:     at android.app.Activity.dispatchKeyEvent(Activity.java:4085)
System.err:     at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.java:122)
System.err:     at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:84)
System.err:     at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:140)
System.err:     at androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent(AppCompatActivity.java:569)
System.err:     at androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent(WindowCallbackWrapper.java:59)
System.err:     at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent(AppCompatDelegateImpl.java:3054)
System.err:     at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:390)
System.err:     at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5947)
System.err:     at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5815)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5310)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5485)
System.err:     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5542)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5518)
System.err:     at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5676)
System.err:     at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3179)
System.err:     at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2721)
System.err:     at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2712)
System.err:     at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3156)
System.err:     at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
System.err:     at android.os.MessageQueue.nativePollOnce(Native Method)
System.err:     at android.os.MessageQueue.next(MessageQueue.java:335)
System.err:     at android.os.Looper.loop(Looper.java:183)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:7656)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by triniwiz** [comment]
Should be good to with `7.0.0-alpha8`

**Comment by salvatorebnt** [comment]
I just upgraded the version..

`
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onTextChanged failed
System.err: Error: java.lang.Exception: Failed resolving constructor for class 'java.lang.String' with 1 parameters. Check the number and type of arguments.
System.err: Primitive types need to be manually wrapped in their respective Object wrappers.
System.err: If you are creating an instance of an inner class, make sure to always provide reference to the outer `this` as the first argument.
System.err:
System.err: StackTrace:
System.err: onTextChanged(file: node_modules\@triniwiz\nativescript-stripe\index.android.js:786:0)
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err:     at com.tns.gen.android.text.TextWatcher.onTextChanged(TextWatcher.java:31)
System.err:     at android.widget.TextView.sendOnTextChanged(TextView.java:10578)
System.err:     at android.widget.TextView.handleTextChanged(TextView.java:10685)
System.err:     at android.widget.TextView$ChangeWatcher.onTextChanged(TextView.java:13568)
System.err:     at android.text.SpannableStringBuilder.sendTextChanged(SpannableStringBuilder.java:1267)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:576)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:507)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:37)
System.err:     at android.text.method.NumberKeyListener.onKeyDown(NumberKeyListener.java:131)
System.err:     at android.widget.TextView.doKeyDown(TextView.java:8466)
System.err:     at android.widget.TextView.onKeyDown(TextView.java:8240)
System.err:     at android.view.KeyEvent.dispatch(KeyEvent.java:2842)
System.err:     at android.view.View.dispatchKeyEvent(View.java:14247)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.widget.ScrollView.dispatchKeyEvent(ScrollView.java:491)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:476)
System.err:     at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1861)
System.err:     at android.app.Activity.dispatchKeyEvent(Activity.java:4085)
System.err:     at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.java:122)
System.err:     at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:84)
System.err:     at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:140)
System.err:     at androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent(AppCompatActivity.java:569)
System.err:     at androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent(WindowCallbackWrapper.java:59)
System.err:     at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent(AppCompatDelegateImpl.java:3054)
System.err:     at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:390)
System.err:     at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5947)
System.err:     at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5815)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5310)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5485)
System.err:     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5542)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5518)
System.err:     at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5676)
System.err:     at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3179)
System.err:     at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2721)
System.err:     at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2712)
System.err:     at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3156)
System.err:     at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
System.err:     at android.os.MessageQueue.nativePollOnce(Native Method)
System.err:     at android.os.MessageQueue.next(MessageQueue.java:335)
System.err:     at android.os.Looper.loop(Looper.java:183)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:7656)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
System.err: Caused by: java.lang.Exception: Failed resolving constructor for class 'java.lang.String' with 1 parameters. Check the number and type of arguments.
System.err: Primitive types need to be manually wrapped in their respective Object wrappers.
System.err: If you are creating an instance of an inner class, make sure to always provide reference to the outer `this` as the first argument.
System.err:     at com.tns.Runtime.resolveConstructorSignature(Runtime.java:1240)
System.err:     ... 76 more
`

**Comment by triniwiz** [comment]
Yeah sorry all done `7.0.0-alpha9`

**Comment by salvatorebnt** [comment]
Thanks @triniwiz for your support..
Just upgraded to 7.0.0-alpha9.. the Expiry input looks ok now.. But I have similar error during the card and cvc input:

**Card input:**
```
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onTextChanged failed
System.err: Error: java.lang.Exception: Failed resolving constructor for class 'java.lang.String' with 1 parameters. Check the number and type of arguments.
System.err: Primitive types need to be manually wrapped in their respective Object wrappers.
System.err: If you are creating an instance of an inner class, make sure to always provide reference to the outer `this` as the first argument.
System.err:
System.err: StackTrace:
System.err: onTextChanged(file: node_modules\@triniwiz\nativescript-stripe\index.android.js:776:0)
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err:     at com.tns.gen.android.text.TextWatcher.onTextChanged(TextWatcher.java:31)
System.err:     at android.widget.TextView.sendOnTextChanged(TextView.java:10578)
System.err:     at android.widget.TextView.handleTextChanged(TextView.java:10685)
System.err:     at android.widget.TextView$ChangeWatcher.onTextChanged(TextView.java:13568)
System.err:     at android.text.SpannableStringBuilder.sendTextChanged(SpannableStringBuilder.java:1267)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:576)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:507)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:37)
System.err:     at android.text.method.NumberKeyListener.onKeyDown(NumberKeyListener.java:131)
System.err:     at android.widget.TextView.doKeyDown(TextView.java:8466)
System.err:     at android.widget.TextView.onKeyDown(TextView.java:8240)
System.err:     at android.view.KeyEvent.dispatch(KeyEvent.java:2842)
System.err:     at android.view.View.dispatchKeyEvent(View.java:14247)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.widget.ScrollView.dispatchKeyEvent(ScrollView.java:491)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:476)
System.err:     at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1861)
System.err:     at android.app.Activity.dispatchKeyEvent(Activity.java:4085)
System.err:     at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.java:122)
System.err:     at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:84)
System.err:     at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:140)
System.err:     at androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent(AppCompatActivity.java:569)
System.err:     at androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent(WindowCallbackWrapper.java:59)
System.err:     at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent(AppCompatDelegateImpl.java:3054)
System.err:     at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:390)
System.err:     at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5947)
System.err:     at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5815)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5310)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5485)
System.err:     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5542)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5518)
System.err:     at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5676)
System.err:     at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3179)
System.err:     at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2721)
System.err:     at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2712)
System.err:     at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3156)
System.err:     at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
System.err:     at android.os.MessageQueue.nativePollOnce(Native Method)
System.err:     at android.os.MessageQueue.next(MessageQueue.java:335)
System.err:     at android.os.Looper.loop(Looper.java:183)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:7656)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
System.err: Caused by: java.lang.Exception: Failed resolving constructor for class 'java.lang.String' with 1 parameters. Check the number and type of arguments.
System.err: Primitive types need to be manually wrapped in their respective Object wrappers.
System.err: If you are creating an instance of an inner class, make sure to always provide reference to the outer `this` as the first argument.
System.err:     at com.tns.Runtime.resolveConstructorSignature(Runtime.java:1240)
System.err:     ... 75 more
```

**CVC input:**
```
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onTextChanged failed
System.err: Error: java.lang.Exception: Failed resolving constructor for class 'java.lang.String' with 1 parameters. Check the number and type of arguments.
System.err: Primitive types need to be manually wrapped in their respective Object wrappers.
System.err: If you are creating an instance of an inner class, make sure to always provide reference to the outer `this` as the first argument.
System.err:
System.err: StackTrace:
System.err: onTextChanged(file: node_modules\@triniwiz\nativescript-stripe\index.android.js:809:0)
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err:     at com.tns.gen.android.text.TextWatcher.onTextChanged(TextWatcher.java:31)
System.err:     at android.widget.TextView.sendOnTextChanged(TextView.java:10578)
System.err:     at android.widget.TextView.handleTextChanged(TextView.java:10685)
System.err:     at android.widget.TextView$ChangeWatcher.onTextChanged(TextView.java:13568)
System.err:     at android.text.SpannableStringBuilder.sendTextChanged(SpannableStringBuilder.java:1267)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:576)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:507)
System.err:     at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:37)
System.err:     at android.text.method.NumberKeyListener.onKeyDown(NumberKeyListener.java:131)
System.err:     at android.widget.TextView.doKeyDown(TextView.java:8466)
System.err:     at android.widget.TextView.onKeyDown(TextView.java:8240)
System.err:     at android.view.KeyEvent.dispatch(KeyEvent.java:2842)
System.err:     at android.view.View.dispatchKeyEvent(View.java:14247)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.widget.ScrollView.dispatchKeyEvent(ScrollView.java:491)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
System.err:     at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:476)
System.err:     at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1861)
System.err:     at android.app.Activity.dispatchKeyEvent(Activity.java:4085)
System.err:     at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.java:122)
System.err:     at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:84)
System.err:     at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:140)
System.err:     at androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent(AppCompatActivity.java:569)
System.err:     at androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent(WindowCallbackWrapper.java:59)
System.err:     at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent(AppCompatDelegateImpl.java:3054)
System.err:     at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:390)
System.err:     at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5947)
System.err:     at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5815)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5310)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5485)
System.err:     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5542)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
System.err:     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
System.err:     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
System.err:     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
System.err:     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5518)
System.err:     at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5676)
System.err:     at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3179)
System.err:     at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2721)
System.err:     at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2712)
System.err:     at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3156)
System.err:     at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
System.err:     at android.os.MessageQueue.nativePollOnce(Native Method)
System.err:     at android.os.MessageQueue.next(MessageQueue.java:335)
System.err:     at android.os.Looper.loop(Looper.java:183)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:7656)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
System.err: Caused by: java.lang.Exception: Failed resolving constructor for class 'java.lang.String' with 1 parameters. Check the number and type of arguments.
System.err: Primitive types need to be manually wrapped in their respective Object wrappers.
System.err: If you are creating an instance of an inner class, make sure to always provide reference to the outer `this` as the first argument.
System.err:     at com.tns.Runtime.resolveConstructorSignature(Runtime.java:1240)
System.err:     ... 76 more
```

**Comment by triniwiz** [comment]
published `7.0.0-alpha10`
